FeedBuilder/frmMain.cs

```diff
@@ -81,18 +83,9 @@ private void frmMain_Load(Object sender, EventArgs e)
 			if (!_argParser.ShowGui) Close();
 		}
 
-		private void InitializeFormSettings()
+		private async Task InitializeFormSettings()

```

Comment:
I see that InitializeFormSettings() is called with the await keyword in a later commit, don't understand why it is made async if it is only called sync